config/initializers/cache_buster.rb

```diff
@@ -6,5 +6,6 @@
   config.x.cache_buster = {
     secret_header: ENV['CACHE_BUSTER_SECRET_HEADER'],
     secret: ENV['CACHE_BUSTER_SECRET'],
+    http_method: ENV['CACHE_BUSTER_HTTP_METHOD'] || 'GET',
```

Comment:
Since we want to deprecate the default `CACHE_BUSTER_SECRET_HEADER`, we may want to raise a deprecation warning here if neither `CACHE_BUSTER_HTTP_METHOD` nor `CACHE_BUSTER_SECRET_HEADER` are set.